LICENSE-3rdparty.csv

```diff
@@ -23,6 +23,13 @@ require,retry,MIT,Copyright 2011 Tim Koschützki Felix Geisendörfer
 require,semver,ISC,Copyright Isaac Z. Schlueter and Contributors
 require,source-map,BSD-3-Clause,Copyright 2009-2011 Mozilla Foundation and contributors
 require,source-map-resolve,MIT,Copyright 2014-2020 Simon Lydell 2019 Jinxiang
+dev,@rollup/plugin-commonjs,MIT,Copyright 2019 RollupJS Plugin Contributors
```

Comment:
These should be in alphabetical order to make the file easier to use. Also, bundlers are usually fairly heavy dependencies and can have a hard time with older versions of Node. Any way we can avoid having them in the main package.json?